test/merkle_patricia_tree_test.exs

```diff
@@ -45,7 +46,7 @@ defmodule MerklePatriciaTreeTest do
     "test/support/ethereum_common_tests/TrieTests/trie#{Atom.to_string(type)}.json"
   end
 
-  def maybe_hex("0x" <> _str=hex_string), do: hex_to_binary(hex_string)
+  def maybe_hex(hex_string = "0x" <> _str), do: hex_to_binary(hex_string)
```

Comment:
Is that right? I'm not sure if this was meant to extract the "0x" or not.

Reply:
@hayesgm Credo was complaining about code consistency. 
https://github.com/rrrene/credo/blob/8b6f1e60ed518692da4d20bad11de3c94d048951/lib/credo/check/consistency/parameter_pattern_matching.ex